diff --git a/pkgs/development/lisp-modules-new/packages.nix b/pkgs/development/lisp-modules-new/packages.nix
index 0a6520df537..bef0cfd88e5 100644
--- a/pkgs/development/lisp-modules-new/packages.nix
+++ b/pkgs/development/lisp-modules-new/packages.nix
@@ -110,34 +110,6 @@ let
     ];
   };
 
-  quri = build-asdf-system {
-    src = pkgs.stdenv.mkDerivation {
-      pname = "patched";
-      version = "source";
-      src =  pkgs.fetchzip {
-        url = "http://beta.quicklisp.org/archive/quri/2021-04-11/quri-20210411-git.tgz";
-        sha256 = "1pkvpiwwhx2fcknr7x47h7036ypkg8xzsskqbl5z315ipfmi8s2m";
-      };
-
-      # fix build with ABCL
-      buildPhase = ''
-        sed -i "s,[#][.](asdf.*,#P\"$out/data/effective_tld_names.dat\")," src/etld.lisp
-      '';
-      installPhase = ''
-        mkdir -pv $out
-        cp -r * $out
-      '';
-    };
-    version = "20210411";
-    pname = "quri";
-    lispLibs = with ql; [
-      alexandria
-      babel
-      cl-utilities
-      split-sequence
-    ];
-  };
-
   jzon = build-asdf-system {
     src = pkgs.fetchzip {
       url = "https://github.com/Zulu-Inuoe/jzon/archive/6b201d4208ac3f9721c461105b282c94139bed29.tar.gz";
