diff --git a/pCloudCC/CMakeLists.txt b/pCloudCC/CMakeLists.txt
index c50c8bc..1c50bd2 100644
--- a/pCloudCC/CMakeLists.txt
+++ b/pCloudCC/CMakeLists.txt
@@ -5,7 +5,7 @@ project(pcloudcc)
 
 set(CMAKE_BUILD_TYPE Release)
 
-set(Boost_USE_STATIC_LIBS ON)
+set(Boost_USE_STATIC_LIBS OFF)
 set(Boost_USE_MULTITHREADED ON)
 unset(Boost_INCLUDE_DIR CACHE)
 unset(Boost_LIBRARY_DIRS CACHE)
@@ -78,5 +78,5 @@ link_directories(${PCLSYNC_PATH} ${MBEDTLS_PATH}/library ${SQLITE3_PATH} ${OVERL
 #endif()
 
 message(STATUS "System CPU: " ${CMAKE_SYSTEM_PROCESSOR})
-install(TARGETS pcloudcc DESTINATION /usr/bin)
-install(TARGETS pcloudcc_lib LIBRARY DESTINATION /usr/lib)
+#install(TARGETS pcloudcc DESTINATION /usr/bin)
+#install(TARGETS pcloudcc_lib LIBRARY DESTINATION /usr/lib)
